aur.go

```diff
@@ -7,7 +7,8 @@ import (
 	"net/url"
 )
 
-const aurURL = "https://aur.archlinux.org/rpc.php?"
+//AurURL is the base string from which the query is built
+var AurURL = "https://aur.archlinux.org/rpc.php?"
```

Comment:
nit: the variable name should be `AURURL`.

Reply:
I'll change it, I had like that for a bit but something in my head was telling me this looks too much like a C `#define`